rebar.config

```diff
@@ -18,10 +18,6 @@
 
 {escript_name, rebar3}.
 {escript_emu_args, "%%! +sbtu +A0\n"}.
-%% escript_incl_extra is for internal rebar-private use only.
-%% Do not use outside rebar. Config interface is not stable.
-{escript_incl_extra, [{"relx/priv/templates/*", "_build/default/lib/"},
-                      {"rebar/priv/templates/*", "_build/default/lib/"}]}.
```

Comment:
Keep these top-level rather than in profiles. If we just build rebar3 with versions like `rebar3 escriptize`, we want things to keep working anyway, regardless of whether we have a bootstrap or nightly profile.

These configuration values allow us to ship the templates that are going to be in `rebar3 new` as a command and removing them will break templating.